framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Input/SelectInput.tsx

```diff
@@ -0,0 +1,134 @@
+		width: '520px',
+		background: 'linear-gradient(180deg, #101c3d 0%, #0c152e 100%)',
+	},
+	dropdownIndicator: {
+		background: 'linear-gradient(180deg, #101c3d 0%, #0c152e 100%)',
+	},
+	options: {
+		background: 'linear-gradient(180deg, #101c3d 0%, #0c152e 100%)',
+		color: '#ffffff',
+		':hover': { background: '#4070f4' },
+	},
+	menu: {
+		background: 'linear-gradient(180deg, #101c3d 0%, #0c152e 100%)',
+		color: '#ffffff',
+		border: 'none',
+		'border-bottom': '1px solid rgba(223, 230, 242, 0.2)',
+		'box-sizing': 'border-box',
+		'border-radius': '3px',
+	},
+	menuList: {
+		background: 'linear-gradient(180deg, #101c3d 0%, #0c152e 100%)',
+		border: 'none',
+		'border-bottom': '1px solid rgba(223, 230, 242, 0.2)',
+		'box-sizing': 'border-box',
+		'border-radius': '3px',
+	},
+	control: {
+		background: 'linear-gradient(180deg, #101c3d 0%, #0c152e 100%)',
+		border: 'none',
+		'border-bottom': '1px solid rgba(223, 230, 242, 0.2)',
+		'box-sizing': 'border-box',
+		'border-radius': '3px',
+	},
+	indicatorsContainer: {
+		border: 'none',
+	},
+	input: {
+		color: '#ffffff',
+	},
+	singleValue: {
+		color: '#ffffff',
+		background: '#254898',
+	},
+	multiValue: {
+		padding: '9px 16px 9px 16px',
+		background: '#254898',
+		borderRadius: '18px',
+		height: '28px',
+		alignItems: 'center',
+	},
+	multiValueLabel: {
+		color: '#ffffff',
+		background: '#254898',
+	},
+};
+
+const SelectInput: React.FC<Props> = props => {
+	const { options } = props;
+	const multi = props.multi ?? false;
+	const [selected, updateSelected] = React.useState(props.selected ?? []);
+	const onSelectHandler = (newValue: any) => {
+		updateSelected(newValue);
+	};
+
+	const onSelect = props.onSelect ?? onSelectHandler;
+
+	return (
+		<span className={styles.select}>
+			<Select
+				closeMenuOnSelect={!multi}
+				isMulti={multi}
+				options={options}
+				value={selected}
+				onChange={onSelect}
+				styles={{
+					// Overriding lib component styles, provided -- the component's default styles
+					container: provided => ({ ...provided, ...customStyles.container }),
+					valueContainer: provided => ({ ...provided, ...customStyles.valueContainer }),
+					option: provided => ({ ...provided, ...customStyles.options }),
+					dropdownIndicator: provided => ({ ...provided, ...customStyles.dropdownIndicator }),
+					menu: provided => ({ ...provided, ...customStyles.menu }),
+					control: provided => ({ ...provided, ...customStyles.control }),
+					menuList: provided => ({ ...provided, ...customStyles.menuList }),
+					input: provided => ({ ...provided, ...customStyles.input }),
+					singleValue: provided => ({ ...provided, ...customStyles.singleValue }),
+					indicatorsContainer: provided => ({ ...provided, ...customStyles.indicatorsContainer }),
+					multiValue: provided => ({ ...provided, ...customStyles.multiValue }),
+					multiValueLabel: provided => ({ ...provided, ...customStyles.singleValue }),
+					indicatorSeparator: _ => ({}),
+				}}
```

Comment:
You can create this custom styling object outside component as it will not be changed per component. Its also documented in similar way in the docs. https://react-select.com/styles#provided-styles-and-state